app/assets/javascripts/subscription.js

```diff
@@ -0,0 +1,59 @@
+$(document).ready(function(){
+  $("#btn_subscribe").click(function (){
+    $("#subscribe").closeModal();
+    var event_id, data;
```

Comment:
**Critical - Code Quality**
Why did you have to define these variables here?
This could be refactored to:
```
  var event_id = $("input[name=sub_event_id]:checkbox:checked").val(),
      data = {
        subscription: {
          event_id:...
        }
      };
```